src/core/checkedint.d

```diff
@@ -682,10 +682,27 @@ unittest
 
 /// ditto
 pragma(inline, true)
+ulong mulu(uint x, ulong y, ref bool overflow)
+{
+    ulong r = x * y;
+    if (y >> 32 && x && (r / x) != y)
```

Comment:
If a condition has 3 clauses, `A && B && C`, then if all 3 are hit then the line count will be `3`. So if the line above has 1 hit, and the conditional has 3, you know all 3 are hit. If the line count is `1`, then you know only the first conditional was hit.
Not the greatest, but still more information than just "was any clause hit".